prqlc/prqlc-parser/src/lexer/mod.rs

```diff
@@ -10,12 +6,15 @@ use chumsky::{
 use serde::{Deserialize, Serialize};
 
 use crate::error::{Error, ErrorSource, Reason, WithErrorInfo};
-use crate::lexer;
 use crate::lexer::lr::{Literal, Token, TokenKind, ValueAndUnit};
 use crate::span::Span;
 
+pub mod lr;
+#[cfg(test)]
+mod test;
```

Comment:
I think convention is to always put `mod` at the top, no?

Reply:
...apparently not. Both RustRover's Optimize Imports and cargo+nightly fmt give this ordering (I just checked)

Reply:
https://doc.rust-lang.org/beta/style-guide/items.html

> use statements, and module declarations (mod foo;, not mod { ... }) must come before other items. Put imports before module declarations.

You're right! Weird, I think most projects I've worked on do the opposite. TIL. Let's do it